.github/problem-matchers/generic.json

```diff
@@ -1,13 +1,13 @@
 {
     "problemMatcher": [
         {
-            "__comment_owner": "match depmod warnings/errors",
-            "owner": "depmod",
+            "__comment_owner": "match generic tool warnings/errors",
+            "owner": "generic",
             "pattern": [
                 {
                     "__comment_regexp1": "depmod: ERROR: Cycle detected: crypto -> crypto_algapi -> crypto",
                     "__comment_regexp2": "depmod: ERROR: Found 2 modules in dependency cycles!",
-                    "regexp": "^depmod:\\s+(FATAL|ERROR|WARNING|NOTICE|INFO|DEBUG):\\s+(.*)$",
+                    "regexp": "^[^:]+:\\s+(FATAL|ERROR|WARNING|NOTICE|INFO|DEBUG):\\s+(.*)$",
```

Comment:
If this doesn't need to be strictly error/warning, how about:

`"^[^:]+:\\s+([^:]+):\\s+(.*)$"`

Though to capture the tool name, I wonder about:

`"(^[^:]+:\\s+([^:]+):\\s+.*)$"`

where message:1 and severity:2 :P

Reply:
I like the second one better because we show the full message, which will make triaging easily :)